src/interceptors/exception.interceptor.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ArgumentsHost,
+  Catch,
+  ExceptionFilter,
+  HttpException,
+  Logger,
+} from '@nestjs/common';
+import { Response } from 'express';
+import { APIResponse } from 'src/utils/apiResponse.utils';
```

Comment:
PullProofAI

The import path for `APIResponse` seems incorrect. Please ensure the correct path is used.
```diff
+import { APIResponse } from '../utils/apiResponse.utils';
```



<!-- This is an auto-generated comment by PullProofAI -->